### ISSUE TITLE
enum {} causes the compiler to hang

### ISSUE THREAD

**Issue by KaleidoDeer** [vlang/v#3519]
**V version:**  0.1.24 d92291d
**OS:** Manjaro Linux

**What did you do?**
```v
module main
enum {
	tes = 3
}

fn main() {
	println("test")
}
```

I was trying to use it as a form of ```enum myenum = 5``` for some bindings but the compiler just hangs and never completes. I'm not sure if "nameless" enums are meant to be supported though.

**Comment by TheoHd** [comment]
Hi,

`enum` is not meant to be nameless, this is what happened when I run your code on v0.1.24 on Windows 10:

```
PS C:\Users\thuchard\v_test> v .\test.v
test.v:2:4: Nameless enums are not allowed.
    1| module main
    2| enum {
          ^
    3|  tes = 3
    4| }
```

But something must be done to handle the compiler hanging on manjaro to replace it by this error.

**Comment by spytheman** [comment]
Fixed in #3520 . @TheoHd try to do a fresh clone from https://github.com/vlang/v/ .
